composer.json

```diff
@@ -40,7 +46,7 @@
 	"require": {
 		"php": "^7.4 || >=8.0",
 		"erusev/parsedown": "^1.7.4",
-		"infinum/eightshift-forms-utils": "^1.2.4"
+		"infinum/eightshift-forms-utils": "1.3.0"
 	},
 	"suggest": {
```

Comment:
This part can be removed. People who need to run tests and do run them probably have either pcov or xdebug installed.